src/V3EmitV.cpp

```diff
@@ -99,6 +99,7 @@ class EmitVBaseVisitor VL_NOT_FINAL : public EmitCBaseVisitor {
         putqs(nodep, "end\n");
     }
     virtual void visit(AstInitialAutomatic* nodep) override { iterateChildren(nodep); }
+    virtual void visit(AstInitialStatic* nodep) override { iterateChildren(nodep); }
```

Comment:
I'm wondering if we should put AstInitialStatic into master.

Reply:
I added this to master, It's semantically different, though on master it's handled the same as `AstInitial` (because it just happens to work due to initial blocks being ordered on maser).

Reply:
Actually this is the one that allows raising errors on assignments to const in initial blocks, so that's a feature as well.